docs/relational_table/storing-state-using-relational-table.md

```diff
@@ -38,7 +38,7 @@ For the detailed schema, please check [doc](relational-table-schema.md)
 
 <!-- Todo: link cconsistence hash doc and state table agg doc -->
 ## Relational Table Layer
-[source code](https://github.com/singularity-data/risingwave/blob/main/src/storage/src/table/state_table.rs)
+[source code](https://github.com/singularity-data/risingwave/blob/4e66ca3d41435c64af26b5e0003258c4f7116822/src/storage/src/table/state_table.rs)
```

Comment:
This files is deleted in #4854. PTAL whether this doc needs to be updated @wcy-fdu. Thanks!